package-lock.json

```diff
@@ -23097,14 +21416,6 @@
       "resolved": "https://registry.npmjs.org/wordwrap/-/wordwrap-0.0.2.tgz",
       "integrity": "sha1-t5Zpu0LstAn4PVg8rVLKF+qhZD8="
     },
-    "worker-farm": {
```

Comment:
preety big package-lock change, are you sure?

Reply:
I think it's because ember-auto-import and its dependencies were removed.

Reply:
Anyway, I have npm 6.1.0 and that's what I got after `npm i` :man_shrugging: